app/controllers/super_admin/installation_configs_controller.rb

```diff
@@ -2,10 +2,15 @@ class SuperAdmin::InstallationConfigsController < SuperAdmin::ApplicationControl
   # Overwrite any of the RESTful controller actions to implement custom behavior
   # For example, you may want to send an email after a foo is updated.
   #
-  # def update
-  #   super
-  #   send_foo_updated_email(requested_resource)
-  # end
+  def update
```

Comment:
lets move this to a `after_commit` callback in the installation config model instead